cli/src/lib/presenter/webPresenter.ts

```diff
@@ -23,6 +23,11 @@ export class WebPresenter extends CsvPresenter {
     app.get("/data/files.csv", respondwithCSV(o => this.writeFiles(o)));
     app.get("/data/diffs.csv", respondwithCSV(o => this.writeDiffs(o)));
     app.get("/data/kmers.csv", respondwithCSV(o => this.writeKmers(o)));
+    app.get("/data/blocks/:id.json", ((req, res) => {
```

Comment:
We'll probably want to return a 404 if the `id` is incorrect.

I suggest creating a mapping between the `Diff`'s and their id's in the web prestenter, so you make this lookup more easily.